ESP32LapTimer/src/Wireless.h

```diff
@@ -0,0 +1,16 @@
+#include <Arduino.h>
+#include <WiFi.h>
+#include <DNSServer.h>
+
+extern byte DNS_PORT;
```

Comment:
Any reason this needs to be visible?

Reply:
I thought there was a reference to it somewhere else but apparently not. Ill change it.

Reply:
Fixed in 6a31ca1